src/js/gi/selectors/calculator.js

```diff
@@ -1083,6 +1061,29 @@ export const getCalculatedBenefits = createSelector(
       calculatedBenefits.outputs.perTerm.yellowRibbon.terms[5].visible = false;
     }
 
+    if (institutionType === 'ojt') {
+      calculatedBenefits.inputs = {
+        ...calculatedBenefits.inputs,
+        tuition: false,
+        books: false,
+        yellowRibbon: false,
+        scholarships: false,
+        tuitionAssist: false,
```

Comment:
Because `tuitionAssist` could previously be set to `true`, this will also now hide it entirely as long as the profile page is for OJT. It seems it should be correct behavior, as tuition shouldn't factor into the benefits for an employer, but awaiting confirmation from #5317.